### ISSUE TITLE
Error : spawn ENAMETOOLONG

### ISSUE THREAD

**Issue by pofider** [jsreport/jsreport#202]
Several people were hit by rendering error ENAMETOOLONG. The error went away after restarting

```
error: Error when processing render request Error during rendering report: spawn ENAMETOOLONG Error: spawn ENAMETOOLONG
    at exports._errnoException (util.js:746:11)
    at ChildProcess.spawn (child_process.js:1162:11)
    at exports.spawn (child_process.js:995:9)
    at Object.exports.fork (child_process.js:581:10)
    at ScriptsManager.execute (E:\jsreport\node_modules\script-manager\lib\manager-processes.js:31:31)
    at module.exports (E:\jsreport\node_modules\toner\lib\engine.js:4:25)
    at E:\jsreport\node_modules\toner\lib\toner.js:70:9
    at ScriptsManager.ensureStarted (E:\jsreport\node_modules\script-manager\lib\manager-processes.js:17:5)
    at Toner.renderEngine (E:\jsreport\node_modules\toner\lib\toner.js:65:24)
```

My suspicion is on this piece of code
https://github.com/pofider/node-script-manager/blob/master/lib/manager-processes.js#L29
Seems we are pushing to execArgv in every rendering request, but never clearing it.

I'll test it now if and if I was right, there will be hotfix on the way.

**Comment by pofider** [comment]
The issue was originally submitted here https://groups.google.com/forum/#!searchin/jsreport/spawn/jsreport/pKJlD_X80ag/3s6B-sjbAwAJ

**Comment by pofider** [comment]
So far I cannot replicate the problem on windows 10. I'll try an older windows on azure.

**Comment by pofider** [comment]
I was right, the problem is now fixed in jsreport@0.13.4

For those who wants a quick fix without updating jsreport...

Change [this](https://github.com/pofider/node-script-manager/blob/0.4.0/lib/manager-processes.js#L29) line to 
```js
if (!_.filter(process.execArgv, function (arg) {
          return !S(arg).startsWith("--expose-gc");
      }).length) {
        process.execArgv.push("--expose-gc");
    }
```
